functests/validation/validation.go

```diff
@@ -0,0 +1,232 @@
+		})
+
+		It("should have the sriov operator deployment in running state", func() {
+			deploy, err := client.Client.Deployments(utils.SriovNamespace).Get(utils.SriovOperatorDeploymentName, metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			Expect(deploy.Status.Replicas).To(Equal(deploy.Status.ReadyReplicas))
+
+			pods, err := client.Client.Pods(utils.SriovNamespace).List(metav1.ListOptions{
+				LabelSelector: fmt.Sprintf("name=%s", utils.SriovOperatorDeploymentName)})
+			Expect(err).ToNot(HaveOccurred())
+
+			Expect(len(pods.Items)).To(Equal(1))
+			Expect(pods.Items[0].Status.Phase).To(Equal(corev1.PodRunning))
+		})
+
+		It("Should have the sriov CRDs available in the cluster", func() {
+			crd := &apiext.CustomResourceDefinition{}
+			err := client.Client.Get(context.TODO(), goclient.ObjectKey{Name: utils.SriovNetworkNodePolicies}, crd)
+			Expect(err).ToNot(HaveOccurred())
+
+			err = client.Client.Get(context.TODO(), goclient.ObjectKey{Name: utils.SriovNetworkNodeStates}, crd)
+			Expect(err).ToNot(HaveOccurred())
+
+			err = client.Client.Get(context.TODO(), goclient.ObjectKey{Name: utils.SriovNetworks}, crd)
+			Expect(err).ToNot(HaveOccurred())
+
+			err = client.Client.Get(context.TODO(), goclient.ObjectKey{Name: utils.SriovOperatorConfigs}, crd)
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		It("should deploy the injector pod if requested", func() {
+			operatorConfig := &sriovv1.SriovOperatorConfig{}
+			err := client.Client.Get(context.TODO(), goclient.ObjectKey{Name: "default", Namespace: utils.SriovNamespace}, operatorConfig)
+			Expect(err).ToNot(HaveOccurred())
+
+			if *operatorConfig.Spec.EnableInjector {
+				daemonset, err := client.Client.DaemonSets(utils.SriovNamespace).Get("network-resources-injector", metav1.GetOptions{})
+				Expect(err).ToNot(HaveOccurred())
+				Expect(daemonset.Status.DesiredNumberScheduled).To(Equal(daemonset.Status.NumberReady))
+			} else {
+				_, err := client.Client.DaemonSets(utils.SriovNamespace).Get("network-resources-injector", metav1.GetOptions{})
+				Expect(err).To(HaveOccurred())
+				Expect(errors.IsNotFound(err)).To(BeTrue())
+			}
+		})
+
+		It("should deploy the operator webhook if requested", func() {
+			operatorConfig := &sriovv1.SriovOperatorConfig{}
+			err := client.Client.Get(context.TODO(), goclient.ObjectKey{Name: "default", Namespace: utils.SriovNamespace}, operatorConfig)
+			Expect(err).ToNot(HaveOccurred())
+
+			if *operatorConfig.Spec.EnableOperatorWebhook {
+				daemonset, err := client.Client.DaemonSets(utils.SriovNamespace).Get("operator-webhook", metav1.GetOptions{})
+				Expect(err).ToNot(HaveOccurred())
+				Expect(daemonset.Status.DesiredNumberScheduled).To(Equal(daemonset.Status.NumberReady))
+			} else {
+				_, err := client.Client.DaemonSets(utils.SriovNamespace).Get("operator-webhook", metav1.GetOptions{})
+				Expect(err).To(HaveOccurred())
+				Expect(errors.IsNotFound(err)).To(BeTrue())
+			}
+		})
+	})
+
+	Context("SCTP", func() {
```

Comment:
Same, lowercase as in the sctp suite. In this way skipping / focusing will work as well.